diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -2082,7 +2082,7 @@ if (current_toolchain == default_toolchain) {
 
   chrome_public_apk_or_module_tmpl("chrome_public_apk") {
     target_type = "android_apk"
-    apk_name = "ChromePublic"
+    apk_name = "HeliumSync"
     if (defined(android_app_secondary_abi)) {
       # Ensure 64-bit chrome does not depend on 32-bit things.
       assert_no_deps =
diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -25,7 +25,7 @@ declare_args() {
   # Android package name to use when compiling the public chrome targets
   # (chrome_public_apk, trichrome_chrome_bundle, etc. as well as the
   # *_bundle variants). This defaults to the Chromium browser package name.
-  chrome_public_manifest_package = "org.chromium.chrome"
+  chrome_public_manifest_package = "computer.helium.sync"
   if (use_stable_package_name_for_trichrome) {
     chrome_public_manifest_package += ".stable"
   } else if (android_channel != "default") {
@@ -33,7 +33,7 @@ declare_args() {
   }
 }
 
-chrome_public_test_manifest_package = "org.chromium.chrome.tests"
+chrome_public_test_manifest_package = "computer.helium.sync.tests"
 
 default_chrome_public_jinja_variables = [
   "channel=$android_channel",
diff --git a/chrome/android/java/res_chromium_base/values/channel_constants.xml b/chrome/android/java/res_chromium_base/values/channel_constants.xml
--- a/chrome/android/java/res_chromium_base/values/channel_constants.xml
+++ b/chrome/android/java/res_chromium_base/values/channel_constants.xml
@@ -7,7 +7,7 @@ found in the LICENSE file.
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Chromium</string>
-    <string name="bookmark_widget_title" translatable="false">Chromium bookmarks</string>
-    <string name="search_widget_title" translatable="false">Chromium search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Chromium quick action search</string>
+    <string name="app_name" translatable="false">Helium Passwords</string>
+    <string name="bookmark_widget_title" translatable="false">Helium Passwords bookmarks</string>
+    <string name="search_widget_title" translatable="false">Helium Passwords search</string>
+    <string name="quick_action_search_widget_title" translatable="false">Helium Passwords quick action search</string>
 </resources>
diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -294,10 +294,10 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </then>
         <else>
           <message name="IDS_PRODUCT_NAME" desc="The Chrome application name" translateable="false">
-            Chromium
+            Helium Passwords
           </message>
           <message name="IDS_SHORT_PRODUCT_NAME" desc="The Chrome application short name." translateable="false">
-            Chromium
+            Helium Passwords
           </message>
         </else>
       </if>
