diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2767,7 +2767,6 @@ static_library("browser") {
       "//chrome/browser/profiles:misc",
       "//chrome/browser/profiles:profiles_extra_parts_impl",
       "//chrome/browser/push_messaging:impl",
-      "//chrome/browser/safe_browsing",
       "//chrome/browser/signin:impl",
       "//chrome/browser/sync",
       "//chrome/browser/task_manager:impl",
diff --git a/chrome/browser/safe_browsing/BUILD.gn b/chrome/browser/safe_browsing/BUILD.gn
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -112,7 +112,7 @@ static_library("safe_browsing") {
   # Sources and dependencies added to this block must not depend on anything
   # requiring safe_browsing_mode != 0.
-  if (is_android) {
+  if (false) {
     deps += [
       "//chrome/browser/password_manager/android:utils",
       "//chrome/browser/ui/android/safe_browsing",
